api/src/main/resources/application.yml

```diff
@@ -94,6 +94,7 @@ card-cancellation:
     minimum-lock-time: "PT1M"
     maximum-lock-time: "PT59M"
 
+report-a-birth-message-delay-duration: P0D
```

Comment:
Should this belong within 'payment-cycle'? Also, to be in keeping with 'change-in-payment-email-delay' we should remove '-duration' from the name.

Reply:
Moved under payment-cycle and removed the -duration part of the name.